### PR TITLE
Change multimedia export to check for duplicates

### PR DESCRIPTION
**Pull request by RynhardPietersen**
## Technical Summary

[This ticket](https://dimagi-dev.atlassian.net/browse/SAAS-13564) contains any relevent explanations.
Some form exports with repeat groups prepare duplicate attachments for download. This should stop the duplicates from downloading. 

### Safety story

Tested locally and currently testing on staging.

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/reports/analytics/esaccessors.py [hunk 1]
```diff
@@ -584,9 +584,13 @@ def get_form_ids_having_multimedia(domain, app_id, xmlns, datespan, user_types):
 
 def media_export_is_too_big(es_query):
     size = 0
+    unique_attachments = []
     for form in _forms_with_attachments(es_query):
         for attachment in form.get('external_blobs', {}).values():
-            size += attachment.get('content_length', 0)
+            attachment_id = attachment.get('id', 0)
```
Comment (biyeun) [inline corehq/apps/reports/analytics/esaccessors.py:590]: i think the default value of `None` should be fine for getting the `id`?

File: corehq/apps/reports/analytics/esaccessors.py [hunk 2]
```diff
@@ -584,9 +584,13 @@ def get_form_ids_having_multimedia(domain, app_id, xmlns, datespan, user_types):
...
+            if attachment_id not in unique_attachments:
```
Comment (biyeun) [inline corehq/apps/reports/analytics/esaccessors.py:591]: would then check to see if `attachment_id` is not `None`

File: corehq/apps/reports/analytics/esaccessors.py [hunk 3]
```diff
@@ -584,9 +584,13 @@ def get_form_ids_having_multimedia(domain, app_id, xmlns, datespan, user_types):
 
 def media_export_is_too_big(es_query):
```
Comment (biyeun) [inline corehq/apps/reports/analytics/esaccessors.py:585]: @RynhardPietersen is it possible to also include tests for this?

File: corehq/apps/reports/tasks.py [hunk 4]
```diff
@@ -296,11 +296,16 @@ def _format_filename(form_info, question_id, extension, case_id_to_name):
 
 def _write_attachments_to_file(fpath, num_forms, forms_info, case_id_to_name):
     total_size = 0
+    unique_attachment_ids = []
```
[the remark this answers is not in the document]
Comment (RynhardPietersen) [reply corehq/apps/reports/tasks.py:299]: Sure, that sounds reasonable.

**Review by AmitPhulera — CHANGES_REQUESTED** [review]

**Review by AmitPhulera — APPROVED** [review]
Great work @RynhardPietersen 🎉

**Review by biyeun — APPROVED** [review]
looks good!

**Pull request merged**